ui/components.d.ts

```diff
@@ -55,8 +51,6 @@ declare module 'vue' {
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
-    NUploadFileList: typeof import('naive-ui')['NUploadFileList']
-    NUploadTrigger: typeof import('naive-ui')['NUploadTrigger']
     NWatermark: typeof import('naive-ui')['NWatermark']
     RouterLink: typeof import('vue-router')['RouterLink']
     RouterView: typeof import('vue-router')['RouterView']
```

Comment:
No significant code differences were identified and no potential issues or optimizations could be found based on the given text.